c/datasetjson.c

```diff
@@ -1602,34 +1655,520 @@ void respondWithDataset(HttpResponse* response, char* absolutePath, int jsonMode
   );
 
   if (daRC != RC_DYNALLOC_OK) {
-    zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_DEBUG,
-    		    "error: ds alloc dsn=\'%44.44s\', member=\'%8.8s\', dd=\'%8.8s\',"
+    zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_SEVERE,
```

Comment:
Do not turn these `ZOWE_LOG_DEBUG` into `ZOWE_LOG_SEVERE` and `ZOWE_LOG_INFO` because they will be printed by default. Since this is library code, we can't print anything here by default.

Reply:
That was the quickest way for me to see the message, rather than enabling ZOWE_LOG_DEBUG.
Yes, these will need tidying before MERGE.